src/keas/kmi/persistent.py

```diff
@@ -14,8 +14,7 @@
 """Encrypted persistent objects
 """
 from __future__ import absolute_import
-import cPickle
-import cStringIO
+from ZODB._compat import BytesIO, Pickler, Unpickler
```

Comment:
Please do not import from private modules of other packages (here, ZODB).  Create a `keas.kmi._compat` module and copy the relevant bits into it -- or just use [`six`](https://pypi.python.org/pypi/six).